src/report-filter-controls.tsx

```diff
@@ -88,92 +88,94 @@ function ReportFilterControls({
   }
 
   return (
-    <div className="usa-form">
-      <form ref={formRef} onChange={update}>
-        <div>
-          <label>
-            Start
-            <input
-              type="date"
-              name="start"
-              value={yearMonthDayFormat(start)}
-              className="usa-input"
-            />
-          </label>
-        </div>
-        <div>
-          <label>
-            Finish
-            <input
-              type="date"
-              name="finish"
-              value={yearMonthDayFormat(finish)}
-              className="usa-input"
-            />
-          </label>
-        </div>
-        <div>
-          <button type="button" className="usa-button" onClick={updateTimeRange(utcWeek, -1)}>
-            &larr; Previous Week
-          </button>
-          <button type="button" className="usa-button" onClick={updateTimeRange(utcWeek, +1)}>
-            Next Week &rarr;
-          </button>
-        </div>
-        <div>
-          <div className="usa-radio">
-            <input
-              type="radio"
-              id="ial-1"
-              name="ial"
-              value="1"
-              checked={ial === 1}
-              className="usa-radio__input"
-            />
-            <label htmlFor="ial-1" className="usa-radio__label">
-              IAL 1
+    <>
+      <div className="usa-form">
+        <form ref={formRef} onChange={update}>
```

Comment:
While we're here: Do we need the `div` wrapper, or can we collapse that to the form?

```suggestion
      <form ref={formRef} onChange={update} className="usa-form">
```

Reply:
great catch! 3676bfb6ed3aaf8a93f4d4afd143e4bf0755fd0c